From 1664d99d27ccb0434e09179513f23a81cc86ab65 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lucie=20G=C3=A9rard?= <lucie.gerard@qt.io>
Date: Thu, 9 Oct 2025 15:48:50 +0200
Subject: [PATCH 433/553] Adapt to reuse version 6

Task-number: QTBUG-140916
Pick-to: 6.8
Change-Id: I5f865a54efc505312ba4172398efea6d78f7a9a3
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
(cherry picked from commit a53be8b935964836b643f1b40d56fe16a35cf9c4)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/3rdparty/iaccessible2/REUSE.toml | 2 +-
 src/3rdparty/libpsl/REUSE.toml       | 2 +-
 util/unicode/main.cpp                | 2 ++
 util/xkbdatagen/main.cpp             | 3 ++-
 4 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/src/3rdparty/iaccessible2/REUSE.toml b/src/3rdparty/iaccessible2/REUSE.toml
index f4d00d7f4f1..24da6e46400 100644
--- a/src/3rdparty/iaccessible2/REUSE.toml
+++ b/src/3rdparty/iaccessible2/REUSE.toml
@@ -5,5 +5,5 @@ path = ["**"]
 precedence = "closest"
 SPDX-FileCopyrightText = ["Copyright (c) 2000, 2006 Sun Microsystems, Inc.",
     "Copyright (c) 2006 IBM Corporation",
-    "Copyright (c) 2007, 2010, 2012, 2013 Linux Foundation\nIAccessible2 is a trademark of the Linux Foundation. The IAccessible2 mark may be used in accordance with the Linux Foundation Trademark Policy to indicate compliance with the IAccessible2 specification."]
+    "Copyright (c) 2007, 2010, 2012, 2013 Linux Foundation IAccessible2 is a trademark of the Linux Foundation. The IAccessible2 mark may be used in accordance with the Linux Foundation Trademark Policy to indicate compliance with the IAccessible2 specification."]
 SPDX-License-Identifier = "BSD-3-Clause"
diff --git a/src/3rdparty/libpsl/REUSE.toml b/src/3rdparty/libpsl/REUSE.toml
index 6c911a98dfe..7b63d4ed725 100644
--- a/src/3rdparty/libpsl/REUSE.toml
+++ b/src/3rdparty/libpsl/REUSE.toml
@@ -10,6 +10,6 @@ SPDX-License-Identifier = "BSD-3-Clause"
 [[annotations]]
 path = ["psl_data.cpp"]
 precedence = "closest"
-SPDX-FileCopyrightText = ["The list was originally provided by Jo Hermans <jo.hermans@gmail.com>.\nIt is now maintained on github (https://github.com/publicsuffix/list)."]
+SPDX-FileCopyrightText = ["The list was originally provided by Jo Hermans <jo.hermans@gmail.com>.It is now maintained on github (https://github.com/publicsuffix/list)."]
 SPDX-License-Identifier = "MPL-2.0"
 
diff --git a/util/unicode/main.cpp b/util/unicode/main.cpp
index ffd7d185905..1f31febeaaf 100644
--- a/util/unicode/main.cpp
+++ b/util/unicode/main.cpp
@@ -3365,11 +3365,13 @@ int main(int, char **)
     QByteArray normalizationCorrections = createNormalizationCorrections();
     QByteArray idnaMapping = createIdnaMapping();
 
+    # REUSE-IgnoreStart
     QByteArray header =
         "// Copyright (C) 2020 The Qt Company Ltd.\n"
         "// SPDX-License-Identifier: Unicode-3.0\n"
         "// Qt-Security score:significant reason:default\n"
         "\n";
+    # REUSE-IgnoreEnd
 
     QByteArray note =
         "/* This file is autogenerated from the Unicode " DATA_VERSION_S " database. Do not edit */\n\n";
diff --git a/util/xkbdatagen/main.cpp b/util/xkbdatagen/main.cpp
index 6dfd765e392..4c9227816b1 100644
--- a/util/xkbdatagen/main.cpp
+++ b/util/xkbdatagen/main.cpp
@@ -374,13 +374,14 @@ int main(int argc, char **argv)
     }
 
     QList<XKBLayout> layouts = findLayouts(layoutList);
-
+    # REUSE-IgnoreStart
     // copyright and stuff
     printf("// Copyright (C) 2016 The Qt Company Ltd.\n"
            "// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only\n"
            "// This file is auto-generated, do not edit!\n"
            "// (Generated using util/xkbdatagen)\n"
            "\n");
+    # REUSE-IgnoreEnd
 
     // data structure
     printf("static struct {\n"
-- 
2.51.2

